src/main.py

```diff
@@ -241,10 +241,17 @@ def main(args):
     parser.add_argument(
         "--amp_level",
         type=str,
-        default="O1",
+        default=None,
```

Comment:
I think the default should be `"O2"` since that's what [PyTorch-Lightning uses as the default](https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html#amp-level).

Reply:
If `None` is changed to `O2`, I believe, there is no way to unset it from the console. Therefore, if the user decides to use `--amp_backend native`, it will crash.

Reply:
Hmmm. If that's the case then `None` is best.